pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -964,6 +967,109 @@ type MSTeamsConfig struct {
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
 }
 
+// RocketChatConfig configures notifications via RocketChat.
+// See https://prometheus.io/docs/alerting/latest/configuration/#rocketchat_config
+type RocketChatConfig struct {
+	// Whether to notify about resolved alerts.
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty" yaml:"send_resolved,omitempty"`
```

Comment:
Please take a look at another existing type, for example [PagerDutyConfig](https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go#L197) it doesn't require the yaml config